test/event_test.py

```diff
@@ -405,6 +405,23 @@ def test_get_type(self):
         for event in q:
             self.assertEqual(event, ev)
 
+    def test_get_exclude(self):
+        pygame.event.post(pygame.event.Event(pygame.USEREVENT))
+        pygame.event.post(pygame.event.Event(pygame.KEYDOWN))
+
+        queue = pygame.event.get(exclude=pygame.KEYDOWN)
+        self.assertEqual(len(queue), 1)
+        self.assertEqual(queue[0].type, pygame.USEREVENT)
+
+        pygame.event.post(pygame.event.Event(pygame.KEYUP))
+        pygame.event.post(pygame.event.Event(pygame.USEREVENT))
+        queue = pygame.event.get(exclude=(pygame.KEYDOWN, pygame.KEYUP))
+        self.assertEqual(len(queue), 1)
+        self.assertEqual(queue[0].type, pygame.USEREVENT)
+
+        queue = pygame.event.get()
+        self.assertEqual(len(queue), 2)
```

Comment:
Can we please have a test for this part from the documentation?
> If  an ``exclude`` parameter is passed, the ``eventtype`` parameter *must* be None.

Testing some other error conditions could be useful.

Reply:
Sure, we can test this.

Theoretically, we can even relax this requirement in a later release (at the expense of code complexity), but in practice, I see no reason to have a type in both the eventtype and the exclude parameter.